packages/api/src/lib/prepareParams.js

```diff
@@ -27,6 +24,64 @@ function isEmpty(obj) {
   return [Object, Array].includes((obj || {}).constructor) && !Object.entries(obj || {}).length;
 }
 
+function processFile(paramName, file) {
```

Comment:
this could use a comment describing roughly what it's doing, and how.

Reply:
I'll do you on better in a different PR and load in the JSDoc ESLint plugin that we have in `oas` and add comments to everything.